contentful_management/client.py

```diff
@@ -317,7 +317,7 @@ def editor_interfaces(self, space_id, content_type_id):
 
         return EditorInterfacesProxy(self, space_id, content_type_id)
 
-    def snapshots(self, space_id, entry_id):
+    def snapshots(self, space_id, resource_id, resource_kind='entries'):
```

Comment:
There are separate proxies for entries/content types, but the Client proxy is the same. As the base class is the same, and interactions for each resource class with the proxies are what's separated.